### PR TITLE
Update greetings.yaml

### PR DESCRIPTION
**Pull request by shivam-sharma7**
typo-fix: issue-message
update: pr-message

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
review/ @AnuragThePathak

File: .github/workflows/greetings.yaml [hunk 1]
```diff
@@ -18,5 +18,5 @@ jobs:
...
-          pr-message: Woah! This is your first Pull Request, Congratulations and thank you for your contribution! 🚀 The team will review it on <u>availability</u> 💫. In the meantime join our <a href="https://discord.io/wemakedevs">discord server</a> and checkout our <a href="https://www.youtube.com/c/KunalKushwaha/">YouTube</a> channel.
+          issue-message: Hey Hey! 👋🏻 Thank you so much for raising an issue. The maintainers will get back to you soon for discussion over the issue! In the meantime join our <a href="https://discord.io/wemakedevs">Discord server</a> and checkout our <a href="https://www.youtube.com/c/KunalKushwaha/">YouTube</a> channel.
+          pr-message: Woah! Thanks to open your first Pull Request in our community, Congratulations 🎉 and thank you for your contribution! 🚀 The team will review it on <u>availability</u> 💫. In the meantime join our <a href="https://discord.io/wemakedevs">Discord server</a> and checkout our <a href="https://www.youtube.com/c/KunalKushwaha/">YouTube</a> channel.
```
Comment (AnuragThePathak) [inline .github/workflows/greetings.yaml:22]: ~~Thanks to open~~ Thanks for opening

File: .github/workflows/greetings.yaml [hunk 2]
```diff
@@ -18,5 +18,5 @@ jobs:
...
-          pr-message: Woah! This is your first Pull Request, Congratulations and thank you for your contribution! 🚀 The team will review it on <u>availability</u> 💫. In the meantime join our <a href="https://discord.io/wemakedevs">discord server</a> and checkout our <a href="https://www.youtube.com/c/KunalKushwaha/">YouTube</a> channel.
+          issue-message: Hey Hey! 👋🏻 Thank you so much for raising an issue. The maintainers will get back to you soon for discussion over the issue! In the meantime join our <a href="https://discord.io/wemakedevs">Discord server</a> and checkout our <a href="https://www.youtube.com/c/KunalKushwaha/">YouTube</a> channel.
+          pr-message: Woah! Thanks for opening your first Pull Request in our community, Congratulations 🎉 and thank you for your contribution! 🚀 The team will review it on <u>availability</u> 💫. In the meantime join our <a href="https://discord.io/wemakedevs">Discord server</a> and checkout our <a href="https://www.youtube.com/c/KunalKushwaha/">YouTube</a> channel.
```
Comment (AnuragThePathak) [inline .github/workflows/greetings.yaml:22]: Revert it to this is your first Pull Request ig. Thanking is repeated in the new message.

**Review by AnuragThePathak — CHANGES_REQUESTED** [review]

**Review by AnuragThePathak — CHANGES_REQUESTED** [review]
Update the Discord link to https://discord.gg/wemakedevs

**Review by AnuragThePathak — APPROVED** [review]

**Pull request merged**